README.md

```diff
@@ -4,6 +4,14 @@ A bot to play planning poker on Discord.
 
 ## Getting Started
 
+### Creating the bot
+* Go to [Discor Developers Portal](https://discord.com/developers/applications).
+* Click on **new application** button in top right.
+* Give the name of the application and navigate to the __Bot__ tab.
+* Click on the **Add Bot** button.
+* Make sure that **Public Bot** is ticked if you want to invite others.
+* Congratulations, You have created your Bot. Configure your Bot accordingly.
```

Comment:
It would be great if "Configure your Bot accordingly" was expanded upon so that there is more info on how to configure the bot.

Reply:
I agree. It should be a little bit more detailed.